clients/banking/src/components/NewMembershipWizard.tsx

```diff
@@ -275,9 +275,16 @@ export const NewMembershipWizard = ({
             },
             () => combineValidators(validateRequired, validateIndividualTaxNumber)(value),
           )
+          .with(
+            {
+              accountCountry: "DEU",
+              residencyAddressCountry: "DEU",
+            },
+            () => combineValidators(validateIndividualTaxNumber)(value),
+          )
           .otherwise(() => undefined);
       },
-      sanitize: value => value.trim(),
+      sanitize: value => value.trim().replaceAll("/", ""),
```

Comment:
`replaceAll` support is OK but might be too "light" https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll

`replace(/\//g, "")` is a bit safer